plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/persistence/SerializableLibraryClasspathContainer.java

```diff
@@ -59,7 +58,8 @@ LibraryClasspathContainer toLibraryClasspathContainer(IJavaProject javaProject,
       classpathEntries.add(entry.toClasspathEntry(baseDirectory, sourceBaseDirectory));
     }
     
-    Library masterLibrary = CloudLibraries.getMasterLibrary(javaProject);
+    Library masterLibrary = new Library(CloudLibraries.MASTER_CONTAINER_ID);
+    masterLibrary.setName("Google APIs"); //$NON-NLS-1$
     if (libraryFiles == null) { // we deserialized an old version
       libraryFiles = new ArrayList<>();
```

Comment:
This field was added in the previous PR (#2409, [code](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/2409/files#diff-22cc39f47cb2fa69d82c706bd981d445R63)) and is used to do `masterLibrary.setLibraryFiles()` just below. Do we still need this?

Reply:
Yes, I think we do. GSON can have set this field to null.

Reply:
Sorry, I was asking if we can remove the field entirely, if it is no longer necessary because we now record library IDs?

Reply:
We do still use the field to record the actual jars (e.g., from the extension points).